src/js/states/profile/edit/edit.module.js

```diff
@@ -51,7 +51,7 @@
 							controller: function( $scope, isLeader ) {
 								$scope.isLeader = isLeader;
 							},
-							template: '<button type="button" class="btn btn-sm btn-danger" ng-if="isLeader" ui-sref="admin">Administrator</button>'
+							template: '<button type="button" class="btn btn-sm btn-danger" ng-if="isLeader" ui-sref="admin" translate>Administrator</button>'
```

Comment:
This doesn't appear to work - "rm -rf src/bower_components; gulp build; gulp onesky" doesn't change the .pot file. What to do?